marlowe-playground-client/src/Marlowe/Linter.purs

```diff
@@ -72,7 +72,8 @@ data Warning
   = NegativePayment IRange
   | NegativeDeposit IRange
   | TimeoutNotIncreasing IRange
-  | UnreachableCase IRange
+  | UnreachableCaseEmptyChoice IRange
+  | UnreachableCaseFalseNotify IRange
```

Comment:
out of scope of this PR but I feel like we should maybe remove `IRange` from `Warning` and use `{pos :: IRange, warning :: Warning }` instead. The `getWarningRange` function annoys me.